src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -160,6 +161,13 @@ public function testWaitUntilSpecificOutput()
         $this->assertEquals("First iteration output\nSecond iteration output\nOne more iteration output\n", $completeOutput);
     }
 
+    public function testWaitUntilReturnspecificOutput()
```

Comment:
I guess the name of this test is not accurate. (It test what happen in case of wrong callback, right?)

Reply:
fixed